tools/rollup-utils.js

```diff
@@ -40,9 +40,13 @@ const knownGlobals = {
 
 export const globals = () => {
   return (moduleId) => {
-      if (knownGlobals[moduleId]) return knownGlobals[moduleId];
-
-      const modulePkg = JSON.parse(readFileSync(require.resolve(join(moduleId, 'package.json'))));
-      return modulePkg.globalName;
+      if (moduleId in knownGlobals) return knownGlobals[moduleId];
+
+      try {
+        const modulePkg = JSON.parse(readFileSync(require.resolve(join(moduleId, 'package.json'))));
+        return modulePkg.globalName;
+      } catch (e) {
+        return null;
+      }
```

Comment:
Is this change related to the new rollup version?

Reply:
Yes. Also, I am not sure that this is not a bug.